src/sass/ResultActionMenu.scss

```diff
@@ -32,7 +32,7 @@ $tooltip-z-index: 1000;
     cursor: pointer;
     display: none;
     position: relative;
-    height: $menu-height;
+    height: $menu-height+1px;
```

Comment:
That seems a bit hackish.

Reply:
Menu Menu Items' top are 1 pixel under the ResultAction Menu's border and both Menu Menu Items and Menu are the same height. Therefore, the Menu Menu Items left border ends 1 pixel under the ResultAction Menu.

I think that it will make more sense to make the Menu Menu Items 1 pixel shorter though, so I'll update that